lib/std/build.zig

```diff
@@ -2309,17 +2313,17 @@ pub const InstallArtifactStep = struct {
             .builder = builder,
             .step = Step.init(.InstallArtifact, builder.fmt("install {}", .{artifact.step.name}), builder.allocator, make),
             .artifact = artifact,
-            .dest_dir = switch (artifact.kind) {
+            .dest_dir = artifact.override_dest_dir orelse switch (artifact.kind) {
                 .Obj => unreachable,
                 .Test => unreachable,
-                .Exe => .Bin,
-                .Lib => .Lib,
+                .Exe => InstallDir{ .Bin = {} },
```

Comment:
I think you're working around a stage1 issue here, but just to let you know it's planned for `.Bin` to coerce to the tagged union here, which would make these kind of code reworkings nicer in the future.

Reply:
Yeah, that seems to be the case unfortunately, I played with it a bit but this was the cleanest it got.